### DISCUSSION TITLE
Working on Brave, but not on TB 91.3

### DISCUSSION THREAD

**Discussion by bkarlan** [darkreader/darkreader#7351] [General]
I've used this before, at least tried it before and had uninstalled it on TB because there were items that wouldn't work right such as context menus's.  It would be dark letters on a black background which was impossible to read.  I just reinstalled it and now it won't do anything on TB.  I have restarted the program.

**Comment by Gusted** [comment]
Please use the issues tab instead, as this isn't a question.
